lib/jekyll/commands/new_theme.rb

```diff
@@ -22,7 +26,8 @@ def process(args, opts)
         raise Jekyll::Errors::InvalidThemeName, "You must specify a theme name."
       end
 
-      new_theme_name = args.join("_")
+      new_theme_name = args.first
+      opts["theme_path"] = args[1..-1].join(" ") if args.size > 1
```

Comment:
Ah, I see the reasoning for the old behaviour now, let me amend this commit :smile:

On 3 Sep 2016 14:25, ashmaroli <notifications@github.com> wrote:

In lib/jekyll/commands/new_theme.rb<https://github.com/jekyll/jekyll/pull/5319#discussion_r77436860>:

> @@ -22,7 +26,8 @@ def process(args, opts)
>          raise Jekyll::Errors::InvalidThemeName, "You must specify a theme name."
>        end
>
> -      new_theme_name = args.join("_")
> +      new_theme_name = args.first
> +      opts["theme_path"] = args[1..-1].join(" ") if args.size > 1


suppose I enter Long Island as the theme_name.. how would this translate as the .gemspec filename?
long_island.gemspec or long island.gemspec?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/jekyll/jekyll/pull/5319/files/b044d013cbc9a73c278750a87300be2bea53dbde#r77436860>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AFi_qaVEynSO5UchQJkhEIiJe5Ywg-45ks5qmXUygaJpZM4J0TET>.